clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -474,6 +475,8 @@ private synchronized void bootstrap() {
       AbstractStorageEngine storageEngine = storageService.getStorageEngine(versionTopic);
       aggVersionedStorageEngineStats.setStorageEngine(versionTopic, storageEngine);
       StoreBackend storeBackend = getStoreOrThrow(storeName);
+      ComplementSet<Integer> persistedPartitionIds = ComplementSet.newSet(storageEngine.getPersistedPartitionIds());
+      storeBackend.unsubscribe(persistedPartitionIds);
```

Comment:
This would unsubscribe every partition on disk, which isn't what we want to happen.

We want to unsubscribe persisted partitions that don't exist in `storeBackend::subscription`